src/w3iProxy/index.ts

```diff
@@ -59,24 +63,20 @@ class Web3InboxProxy {
       projectId: this.projectId
     })
 
-    console.log('this.chatProvider', this.chatProvider)
-    console.log('this.pushProvider', this.pushProvider)
-
+    // Not disabling chatClient as it manages the account
     if (this.chatProvider === 'internal') {
```

Comment:
Don't we need the additional check as for pushClient ?
```suggestion
    if (this.chatProvider === 'internal' && this.uiEnabled.chat) {
```

Reply:
Same Q 👍

Reply:
I guess the comment above isn't clear enough but since chat manages account, we can't not init it even if chat ui is disabled, this will be changed when/if we make a separate auth provider